EduVPN/Helpers/CertificateExpiryHelper.swift

```diff
@@ -160,8 +156,10 @@ extension CertificateExpiryHelper.CertificateStatus {
 
     var shouldShowRenewSessionButton: Bool {
         switch self {
-        case .validFor(_, let proportionRemaining):
-            return proportionRemaining < 0.2
+        case .validFor(let timeRemaining):
+            // Show renewal button if session expires in
+            // less than a week.
+            return timeRemaining < (60 * 24 * 7)
```

Comment:
Calculates minutes, but timeinterval is in seconds.

Reply:
Fixed in PR #406. This code change is no longer part of this PR.